### DISCUSSION TITLE
Change the value of ModelVariables

### DISCUSSION THREAD

**Discussion by Amin-Rzh** [NTNU-IHB/FMI4j#143] [Q&A]
Is it possible to change the value of `ModelVariables ` with  `causality=INPUT` after loading an `FMU` from a file?

**Comment by markaren** [comment]
No, you would need to re-export the model or edit the XML. The latter would not be advised.

**Reply by Amin-Rzh** [reply]
Thanks

**Reply by Amin-Rzh** [reply]
By which module and funtion we can re-export the model?

**Reply by markaren** [reply]
The original author would need to change it.
